flow/otel_metrics/otel_manager.go

```diff
@@ -42,7 +48,52 @@ func setupGrpcOtelMetricsExporter() (sdkmetric.Exporter, error) {
 	return otlpmetricgrpc.New(context.Background())
 }
 
-func SetupOtelMetricsExporter(otelServiceName string) (*sdkmetric.MeterProvider, error) {
+func temporalMetricsFilteringView() sdkmetric.View {
+	exportListString := peerdbenv.GetEnvString("PEERDB_TEMPORAL_OTEL_METRICS_EXPORT_LIST", "")
+	// Special case for exporting all metrics
+	if exportListString == "__ALL__" {
```

Comment:
can we log to say something like we are exposing XYZ metrics

Reply:
I think I can just log the env var, that should be good enough

Reply:
Added log above